libs/pubsub/query/query.peg

```diff
@@ -15,7 +15,7 @@ condition <- tag ' '* (le ' '* (number / time / date)
                       )
 
 tag <- < (![ \t\n\r\\()"'=><] .)+ >
-value <- < '\'' (!["'] .)* '\''>
+value <- < '\'' (('\\' .) / (!['] .))* '\''>
```

Comment:
```
value <- < '\'' (!['] .)* '\''>
```
Why not this?

Reply:
We need to support Go escape character such as `\"` because of JSON unmarshaled string.

Reply:
In this case, how about limiting the escape character to `"`?
```suggestion
value <- < '\'' (('\\' '\"') / (!['] .))* '\''>
```

Reply:
`\"` is one byte data, but you mean data to unmarshal is two bytes data?

Reply:
@ulbqb @Mdaiki0730 sorry for typo in my previous answer. We need to accept any character if it's escaped.

Reply:
I will merge this PR after @ulbqb approve

Reply:
Actual raw data to unmarshal doesn't contain `\` as escape symbol even if it contains `"` as text. 
Previously, `\` as text is not allowed so your PR contains:

- allow `\` as text
- allow `"` as text

Is this correct?

Reply:
> Actual raw data to unmarshal doesn't contain \ as escape symbol even if it contains " as text.
Yes, that's right
> Your PR contains:
Yes, that's correct. So if you want to add `'` in event string. you need to put `\\'` instead of `'`.